services/datamanager/builtin/builtin.go

```diff
@@ -610,28 +614,82 @@ func (svc *builtIn) Reconfigure(
 	if svc.syncSensor != syncSensor {
 		svc.syncSensor = syncSensor
 	}
-
 	syncConfigUpdated := svc.syncDisabled != svcConfig.ScheduledSyncDisabled || svc.syncIntervalMins != svcConfig.SyncIntervalMins ||
 		!reflect.DeepEqual(svc.tags, svcConfig.Tags) || svc.fileLastModifiedMillis != fileLastModifiedMillis ||
 		svc.maxSyncThreads != newMaxSyncThreadValue
 
 	if syncConfigUpdated {
+		svc.syncConfigUpdated = syncConfigUpdated
 		svc.syncDisabled = svcConfig.ScheduledSyncDisabled
 		svc.syncIntervalMins = svcConfig.SyncIntervalMins
 		svc.tags = svcConfig.Tags
 		svc.fileLastModifiedMillis = fileLastModifiedMillis
 		svc.maxSyncThreads = newMaxSyncThreadValue
+	}
+
+	// if datacapture is enabled, kick off a go routine to check if disk space is filling due to
+	// cached datacapture files
```

Comment:
nit, this code doesn't only check, it also deletes files in that case

```suggestion
	// if datacapture is enabled, kick off a go routine to handle disk space filling due to
	// cached datacapture files
```